orchagent/portsorch.cpp

```diff
@@ -3983,3 +3997,22 @@ void PortsOrch::getPortSerdesVal(const std::string& val_str,
         lane_values.push_back(lane_val);
     }
 }
+
+void PortsOrch::flushFDBEntries(sai_object_id_t bridge_port_id)
```

Comment:
Why is `flushFDBEntries `in `portsorch`? IMO, this should be handled in `fdborch`

Reply:
I agree. I cherrypicked these changes. Is it ok if I handle it in next PR or you insist on doing it in this PR?

Reply:
I think it is better to handle it in this PR itself. The functionality doesn't look fit correctly in `portsorch`